includes/class-sensei-quiz.php

```diff
@@ -103,6 +103,28 @@ public function __construct( $file = __FILE__ ) {
 			// Create the quiz progress when the student visits the quiz page for the first time.
 			add_action( 'wp', array( $this, 'maybe_create_quiz_progress' ) );
 		}
+
+		// Enqueue frontend styles.
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_styles' ) );
+	}
+
+	/**
+	 * Enqueue frontend CSS files for quizzes.
+	 *
+	 * @internal
+	 *
+	 * @since  $$next-version$$
+	 * @return void
+	 */
+	public function enqueue_styles() {
+		if (
+			is_admin() ||
```

Comment:
Nit: Do we need the `is_admin` check? I think `wp_enqueue_scripts` is a front-end-only hook.